### DISCUSSION TITLE
Does new home page roll out to you guys yet?

### DISCUSSION THREAD

**Discussion by randomguyingithub** [anthony1x6000/ROBLOX2016stylus#45] [Polls]
![msedge_2404131244_dAfgQ](https://github.com/anthony1x6000/ROBLOX2016stylus/assets/13596387/addbffe4-0386-40a3-b638-48ae45ad0f2a)

**Comment by anthony1x6000** [comment]
Yeah, I have the same layout. I'll get working on updating the style.
Crazy, they got rid of the carousel arrows too

**Reply by anthony1x6000** [reply]
Should be fixed in [release 1.2.9](https://github.com/anthony1x6000/ROBLOX2016stylus/releases/tag/1.2.9).
